nnvm/src/top/vision/resize.h

```diff
@@ -0,0 +1,47 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file resize.h
+ */
+#ifndef NNVM_TOP_VISION_RESIZE_H_
+#define NNVM_TOP_VISION_RESIZE_H_
+
+#include <string>
+#include <vector>
+#include <utility>
+#include <iostream>
+#include <sstream>
+
+namespace nnvm {
+namespace top {
+
+struct ResizeParam : public dmlc::Parameter<ResizeParam> {
+  TShape out_size;
+  std::string layout;
+  std::string mode;
+  bool align_corners;
+
+  DMLC_DECLARE_PARAMETER(ResizeParam) {
+    DMLC_DECLARE_FIELD(out_size)
+      .describe("Scale output size."
+                "actual out size if is_factor false"
```

Comment:
what is is_factor?

Reply:
Left over description from previous changes. Corrected now.